swizzle/add-nodes.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: install ansible prerequisutess
+  import_playbook: ../ansible/pre.yml
+  tags:
+    - prereqs
+    - ansible-prereqs
+
+- name: create a kubeadm token
+  hosts: primary_master
+  tasks:
+    - name: generate a kubeadm token
```

Comment:
Is it worth adding `no_log: True` on this task?

Reply:
Can add that, but afaik this is not leaking a token currently...assuming that is the concern.

Reply:
That was the concern yeah. Seems unnecessary if we are not leaking the token though.